Examples/CMakeLists.txt

```diff
@@ -12,13 +12,21 @@ endforeach()
 
 if(NOT CMAKE_PROJECT_NAME STREQUAL "SimpleITK")
   find_package(SimpleITK REQUIRED)
-
 endif()
 
 option(SimpleITK_TESTING_NOSHOW "Disable the Show function in SimpleITK testing." ON)
 mark_as_advanced(SimpleITK_TESTING_NOSHOW)
 
-# Add individual cxx executables
+
+if(SimpleITK_USE_ELASTIX)
+  # Add individual cxx executables
+  add_executable ( elx elx.cxx )
```

Comment:
@N-Dekker These examples will need to be better names, and integrated into the Sphinx Docs. Ideally the should follow other where there is a directory for the example which contains the code in a couple of languages and a RST file for the Sphinx docs.   Would you like to address that here or should I create a GH issue for it?

Reply:
@blowekamp Thanks for asking. Ideally I would like to have this pull request "as close as possible" to the original work by Kasper (@kaspermarstal) at https://github.com/SuperElastix/SimpleElastix and then have possible further improvements in separate issues and pull requests. Only if you consider this "a show stopper" for the pull request as a whole, we would have to address it here with this PR. So if it's OK to you, I would prefer a separate GH issue in this case.